app/players/elves/druid.py

```diff
@@ -0,0 +1,25 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 favourite_spell: str) -> None:
+
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def get_rating(self) -> int:
+        return len(self._favourite_spell)
+
+    def player_info(self) -> str:
+        return (
+            f"Druid {self.nickname}. "
+            f"{self.nickname} has a favourite spell: "
+            f"{self._favourite_spell}"
+        )
+
+    def play_elf_song(self) -> str:
```

Comment:
The `play_elf_song` method is defined to return a `str`, but it currently does not return any value. Consider returning a string or changing the return type to `None`.